### PR TITLE
Playback Windows Support

### PR DESCRIPTION
**Pull request by tsvetilian-ty**
## Changelog:

* Updated electron-prebuilt to the latest version
* Checks to find if user is on windows or mac and change the titlebar and the window frame, depending on that.
* Add Windows icon
* Updated contribution information and readme file with new information
* [Windows Build](https://github.com/tsvetilian-ty/playback/releases/tag/v1.3.1)

windows-screenshot

### CODE REVIEW

**Comment by iadj** [comment]
Good start!
It seems that protocol support isn't working yet on Windows (opening a magnet:// or playback:// URL from the browser with playback.exe just starts the app without the stream starting), as referenced in: https://github.com/mafintosh/playback/issues/40

**Comment by finnp** [comment]
This is awesome :+1:

**Comment by freeall** [comment]
I'm not on Windows so I can't check that this works, but otherwise it lgtm. @mafintosh should we merge?

File: app.js [hunk 1]
```diff
@@ -24,12 +24,18 @@ var onopen = function (e, lnk) {
 app.on('open-file', onopen)
 app.on('open-url', onopen)
 
+if (process.platform != 'win32'){
+    var frame = false
+}else{
+    var frame = true
+}
+
```
Comment (mafintosh) [inline app.js]: This should just be `var frame = process.platform === 'win32'`

File: index.js [hunk 2]
```diff
@@ -52,7 +52,10 @@ $(document).on('paste', function (e) {
 
 var media = player($('#player')[0])
 var list = playlist()
-titlebar.appendTo('#titlebar')
+
+if (process.platform != 'win32'){
```
Comment (mafintosh) [inline index.js]: could you make this `!==` instead?

**Comment by mafintosh** [comment]
Added a few comments. Other than that this is a great start to add full Windows support so LGTM!

**Comment by iadj** [comment]
@freeall It works fine on Windows, except for the protocol support. I also personally prefer the custom layout rather than the standard Windows one, which this one has. That's subjective though.

File: README.md [hunk 3]
```diff
@@ -17,6 +25,14 @@ If you think it is missing a feature or you've found a bug feel free to open an
 Simply clone this repo and run `npm install`.
 Afterwards you can run `npm start` to run the app.
 
+### To build on Windows
+
+To build on Windows machine open cmd.exe and navigate to project root directory, then run the following command:
+
+`electron-packager ./ Playback --platform=win32 --arch=ia32 --version=0.26.1 --out=release --icon=./icon.ico --ignore=node_modules/electron-*`
```
Comment (sindresorhus) [inline README.md]: This should probably be a npm run script, like the existing OS X bundling.

File: index.js [hunk 4]
```diff
@@ -52,7 +52,10 @@ $(document).on('paste', function (e) {
 
 var media = player($('#player')[0])
 var list = playlist()
-titlebar.appendTo('#titlebar')
+
+if (process.platform !== 'win32'){
```
Comment (sindresorhus) [inline index.js]: `if (process.platform !== 'win32') {`

**Comment by tsvetilian-ty** [comment]
I did the recommended changes @mafintosh

@iadj I changed the frame value to build more native feeling in Windows user

File: package.json [hunk 5]
```diff
@@ -25,7 +25,7 @@
   },
   "devDependencies": {
     "electron-packager": "^3.1.0",
-    "electron-prebuilt": "^0.25.2"
+    "electron-prebuilt": "^0.26.1"
```
Comment (sindresorhus) [inline package.json]: `0.27.2` is now the latest.

**Comment by mafintosh** [comment]
@tsvetilian-ty perfect thanks! could i get you to squash this down to a single commit?

File: package.json [hunk 6]
```diff
@@ -33,7 +33,8 @@
   "scripts": {
     "start": "electron app.js",
     "dev": "electron app.js test.mp4",
-    "bundle": "electron-packager . Playback --ignore=test.mp4 --ignore=node_modules/electron-prebuilt && cp info.plist Playback.app/Contents/Info.plist && cp icon.icns Playback.app/Contents/Resources/atom.icns"
+    "bundle": "electron-packager . Playback --ignore=test.mp4 --ignore=node_modules/electron-prebuilt && cp info.plist Playback.app/Contents/Info.plist && cp icon.icns Playback.app/Contents/Resources/atom.icns",
+    "win-bundle": "electron-packager ./ Playback --platform=win32 --arch=ia32 --version=0.27.2 --out=./ --icon=./icon.ico --ignore=node_modules/electron-*"
```
Comment (sindresorhus) [inline package.json]: `--ignore` is moot as it's the default to ignore `electron`. Same with `--out`

Should be:

```
"win-bundle": "electron-packager . Playback --platform=win32 --arch=ia32 --version=0.27.2 --icon=icon.ico"
```

**Comment by tsvetilian-ty** [comment]
Done @mafintosh

**Comment by mafintosh** [comment]
@tsvetilian-ty NICE! merged! now we just need someone to do a windows build so we can release it *hint* *hint*

**Comment by tsvetilian-ty** [comment]
@mafintosh [Windows build](https://github.com/tsvetilian-ty/playback/releases/tag/v1.3.1) is already done :+1:

**Comment by mafintosh** [comment]
@tsvetilian-ty thats awesome. will publish as soon as i get to decent non-airport internet :)

**Comment by iadj** [comment]
Should add a note to the Windows build that protocols aren't yet supported

File: README.md [hunk 7]
```diff
@@ -8,7 +9,14 @@ In addition to that it supports live streaming of video using http links, BitTor
 ## Installation
 
 To install it download the [latest release](https://github.com/mafintosh/playback/releases/latest) for your platform.
-Currently only builds for MacOSX are available but pull requests are welcome that adds builds for other platforms.
+
+##Currently supported releases:
+
+* OS X
+* Windows
```
Comment (LestaD) [inline README.md]: Win32 only?

**Pull request merged**